kornia/color/colormap.py

```diff
@@ -1,126 +1,121 @@
 from abc import ABC
-from typing import List, Optional
+from typing import List, Optional, Union
 
 import torch
 from torch.nn.functional import interpolate
 
+from kornia.color import colormap_data as cm_data
 from kornia.core import Module, Tensor, tensor
 from kornia.core.check import KORNIA_CHECK_IS_GRAY
 
-RGBColor = List[float]
+RGBColor = cm_data.RGBColor
+
+colormap_options = {
```

Comment:
i think it can go on the init of the class

Reply:
and i would have somehow defined as a mapping with keys as  Enum instead of string to be less error prone. With new syntaxes you can initialize enums from strings for `rgb = ColorPalette["autumn"]`

Reply:
oh, it's true, +1 enums

Reply:
How do I have to do it to use Enum?